docs/installation.rst

```diff
@@ -184,22 +184,19 @@ check the link to install JAX with the most up-to-date recommendations from the
 
     conda create --name desc-env 'python==3.11'
     conda activate desc-env
-    pip install -U "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+    pip install -U "jax[cuda12]"
 
 Then, install DESC,
 
 .. code-block:: sh
 
     git clone https://github.com/PlasmaControl/DESC.git
     cd DESC
-    # remove the jax lines from requirements.txt, as we already have installed them above
-    sed -i '/jax/d' ./requirements.txt
-    # then install as usual
     pip install --editable .
     # optionally install developer requirements (if you want to run tests)
     pip install -r devtools/dev-requirements.txt
 
-Tested and confirmed to work on the Della and Stellar clusters at Princeton as of June 20, 2024.
+Tested and confirmed to work on the Della and Stellar clusters at Princeton as of October 23, 2024.
 
 
 RAVEN (IPP, Germany)
```

Comment:
RAVEN instructions use the jax version that has the `scipy` problem. If it works for them, we can bump up the JAX version and delete the messy scipy instructions

Reply:
I think maybe someone else (@Mandelbr0t or @IssraAli possibly?) had done this check of installation for us in the past? I don't believe I have access to RAVEN anymore

Reply:
Rahul and I made those instructions for Raven in August and they are working for me. The jax version is 0.4.31. I could try today with a newer jax version if you'd like

Reply:
@IssraAli I guess the instructions are outdated then. They install JAX `0.4.23` specifically. If you can use `0.4.31` then the instructions for `scipy` are redundant.